src/cysignals/macros.h

```diff
@@ -262,7 +278,16 @@ static inline void sig_retry(void)
     if (unlikely(cysigs.sig_on_count <= 0))
     {
         fprintf(stderr, "sig_retry() without sig_on()\n");
+
+#ifndef POSIX
```

Comment:
Would `SIGFPE` work also on POSIX?

Reply:
Given that on posix `abort()` do nothing more than a `raise(SIGABRT)` and that `SIGFPE` and `SIGABRT` have the same default action ([see signal.h](http://pubs.opengroup.org/onlinepubs/009696699/basedefs/signal.h.html)) i think `raise(SIGFPE) ` work on POSIX.